### PR TITLE
Add components & links

### PR DESCRIPTION
**Pull request by Peter1907**
Create components (Books, Book, Navbar & Form)
Using routing to display the original page(Books component)
Adding preliminary styling

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = (props) => {
+  const { title, author, genre } = props;
```
Comment (AmaduKamara) [inline src/components/Book.js]: [ Suggested Changes ] 
You're doing great man 👍 

You could destructure the props right in the Book component's parameter.
Hint:
```mdx
const Book = ({ title, author, genre }) => { // }
```
You could do it that way and directly use them in your JSX 👍 
That will make your code lean and clean. Try it and see please

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢🟢🟢 

Hi @Peter1907,

You are doing great @Peter1907, your speed is so high and I like that😂  
Your project is complete!  

ClappingLeonardoDicaprioGIF (3)

## Highlights:
✅ You used Gitflow correctly
✅ Linter checks passed
✅ Good descriptive PR
✅ Initialized project with create-react-app and all components and routes work fine as expected 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Book.js (hunk 1 above)
Comment (Peter1907) [reply src/components/Book.js]: Thank you for the tip ... Yes I saw this once before but I completely forgot about it:joy::joy:
I'll be using that from now on:ok_hand:

Comment (AmaduKamara) [reply src/components/Book.js]: Great 👍

**Pull request merged**